src/Features/LanguageServer/Protocol/Features/Options/CompletionOptionsStorage.cs

```diff
@@ -32,29 +32,29 @@ public static CompletionOptions GetCompletionOptions(this IGlobalOptionService o
-    public static readonly Option2<bool> ShowNewSnippetExperienceFeatureFlag = new("CompletionOptions_ShowNewSnippetExperienceFeatureFlag", CompletionOptions.Default.ShowNewSnippetExperienceFeatureFlag);
-    public static readonly PerLanguageOption2<bool> HideAdvancedMembers = new("CompletionOptions_HideAdvancedMembers", CompletionOptions.Default.HideAdvancedMembers);
-    public static readonly PerLanguageOption2<bool> TriggerOnTyping = new("CompletionOptions_TriggerOnTyping", CompletionOptions.Default.TriggerOnTyping);
-    public static readonly PerLanguageOption2<bool> TriggerOnTypingLetters = new("CompletionOptions_TriggerOnTypingLetters", CompletionOptions.Default.TriggerOnTypingLetters);
-    public static readonly PerLanguageOption2<bool?> TriggerOnDeletion = new("CompletionOptions_TriggerOnDeletion", CompletionOptions.Default.TriggerOnDeletion);
-    public static readonly PerLanguageOption2<EnterKeyRule> EnterKeyBehavior = new("CompletionOptions_EnterKeyBehavior", CompletionOptions.Default.EnterKeyBehavior, serializer: EditorConfigValueSerializer.CreateSerializerForEnum<EnterKeyRule>());
-    public static readonly PerLanguageOption2<SnippetsRule> SnippetsBehavior = new("CompletionOptions_SnippetsBehavior", CompletionOptions.Default.SnippetsBehavior, serializer: EditorConfigValueSerializer.CreateSerializerForEnum<SnippetsRule>());
-    public static readonly PerLanguageOption2<bool> ShowNameSuggestions = new("CompletionOptions_ShowNameSuggestions", CompletionOptions.Default.ShowNameSuggestions);
+    public static readonly Option2<bool> UnnamedSymbolCompletionDisabledFeatureFlag = new("csharp_disable_unnamed_symbol_completion", CompletionOptions.Default.UnnamedSymbolCompletionDisabled);
+    public static readonly Option2<bool> ShowNewSnippetExperienceFeatureFlag = new("csharp_show_new_snippet_experience_feature_flag", CompletionOptions.Default.ShowNewSnippetExperienceFeatureFlag);
+    public static readonly PerLanguageOption2<bool> HideAdvancedMembers = new("dotnet_hide_advanced_members_in_completion", CompletionOptions.Default.HideAdvancedMembers);
+    public static readonly PerLanguageOption2<bool> TriggerOnTyping = new("dotnet_trigger_completion_on_typing", CompletionOptions.Default.TriggerOnTyping);
+    public static readonly PerLanguageOption2<bool> TriggerOnTypingLetters = new("dotnet_trigger_completion_on_typing_letters", CompletionOptions.Default.TriggerOnTypingLetters);
+    public static readonly PerLanguageOption2<bool?> TriggerOnDeletion = new("dotnet_trigger_completion_on_deletion", CompletionOptions.Default.TriggerOnDeletion);
+    public static readonly PerLanguageOption2<EnterKeyRule> EnterKeyBehavior = new("dotnet_completion_enter_key_behavior", CompletionOptions.Default.EnterKeyBehavior, serializer: EditorConfigValueSerializer.CreateSerializerForEnum<EnterKeyRule>());
```

Comment:
>dotnet_completion_enter_key_behavior [](http://example.com/codeflow?start=84&length=36)

`dotnet_return_key_completion_behavior` (I believe we use "return" instead of "enter" in other option names).